tests/e2e_tests/e2e_test_data.json

```diff
@@ -33,12 +33,10 @@
                 "UUID": "dattarro"
             }
         ],
-        "obligatory_fields": [
-            "position",
-            "name",
-            "accessible_by",
-            "type_of_place",
-            "UUID"
+        "location_obligatory_fields": [
+            ["name", "str"],
+            ["accessible_by", "list"],
+            ["type_of_place", "str"]
```

Comment:
It would be great if we had way of testing both configurations and databases:
- feature flag on + db dependent data (as in comment) - checking if feature works
- feature flag off + db without data -  checking if compatible bacwardly

Reply:
Issue for this was created:
https://github.com/Problematy/goodmap/issues/174